<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormParameters</class>
 <widget class="QMainWindow" name="FormParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор параметров</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Набор:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="boxParameterSets">
        <item>
         <property name="text">
          <string>vostok_1</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddSet">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>Отменить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="btnAccept">
      <property name="text">
       <string>Применить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btnRemove">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>199</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QTableView" name="tblParameters">
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btnAdd">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Параметры</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>Добавить</string>
     </property>
     <addaction name="actionAddSet"/>
     <addaction name="action_3"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="actionSaveTemplate"/>
   </widget>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAddSet">
   <property name="text">
    <string>Набор</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Параметр</string>
   </property>
  </action>
  <action name="actionSaveTemplate">
   <property name="text">
    <string>Сохранить как шаблон</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
